index.html

```diff
@@ -1,40 +1,70 @@
+
+            <div class="header-container">
+                <div class="dropdown">
+                    <p class="logo">EZ-Tip</p>
+                    <i class="fas fa-bars"></i>
+                    <div class="dropdown-content dropdown-hidden">
+                        <a class="dropdown-link" href="" target="_blank">Home</a>
+                        <a class="dropdown-link" href="" target="_blank">About</a>
+                        <a class="dropdown-link" href="" target="_blank">Team</a>
+                    </div>
+                    </div>
+                </div>    
+            </div> <!-- End of Header Container -->
+
+        </div><!-- End of Header Background-->
+
+        <div class="middle-img">
+            <img src="img/example-img.jpeg">
+        </div>
+            
+        <div class="background cta">
+            <div class="cta-container">
+                    <div class="cta-main">
+                        <h1>Tip</h1>
+                        <h2>Server did a good job?</h2>
+                        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda ad omnis error dignissimos a magnam repellat dolorum, consequatur mollitia quae, consectetur blanditiis eveniet excepturi similique, voluptate quis esse quasi dolor.</p>
+                        <button>Give Tip</button>
+                        
+                        <h1>Serving</h1>
+                        <h2>Serving at a restaurant</h2> 
+                        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis, totam hic nobis ipsa quis quasi maiores fuga, non ex modi ut earum repellat soluta obcaecati harum. Reprehenderit fuga asperiores voluptas.</p>
+                        <button>Receive Tips</button>
+
+                    </div>
+                    <div class="cta-img">
+                        <img>
+                    </div>
+            </div> <!-- End of CTA Container -->
+        </div><!-- End of CTA Background -->      
+
         <div class="background middle">
-            <h1>Something also Good</h1>
-            <img>
-            <h2>A good header related to Something Good</h2>
+            
         </div>
     </body>
 
     <footer>
-        <div class="nav-footer">
-            <a class="footer-link" href="" target="_blank">Home</a>
-            <a class="footer-link" href="" target="_blank">About</a>
-            <a class="footer-link" href="" target="_blank">Team</a>
+        <div class="footer-container">
+                <div class="nav-footer">
+                        <a class="footer-link" href="" target="_blank">Home</a>
+                        <a class="footer-link" href="" target="_blank">About</a>
+                        <a class="footer-link" href="" target="_blank">Team</a>
+                    </div>
+                    <p class="copyright">Copyright: 2018 Tipease</p>
         </div>
-        <p class="copyright">Copyright: 2018 Tipease</p>
+        
```

Comment:
Good, concise markup - and excellent use of comments for readability.  
https://github.com/eztip/eztip-UI-joe-pak/blob/cb9dc3959d28ed76a1179a968a459628a31183bc/index.html#L14-L70